app/controllers/discussions_controller.rb

```diff
@@ -5,7 +5,7 @@ class DiscussionsController < ApplicationController
   def index
     discussions = Discussion.all
 
-    render json: discussions, each_serializer: DiscussionCompactSerializer
```

Comment:
Like I'm saying on the description, this is not the ideal solution, but it allows us to reduce the number of requests because we can change between index and show without having to make a sync request every time before showing. Also, when we create a discussion we don't need to make a second request to discussions/index in order to get the rest of the information. The applies for the update.